### DISCUSSION TITLE
Aspect Ratio

### DISCUSSION THREAD

**Discussion by pietrop** [vidstack/player#690] [Q&A] [answered]
_Non urgent question._

To set the [aspect ratio](https://www.vidstack.io/docs/player/react/components/ui/aspect-ratio/), Is it recommended to use `style="aspect-ratio: 16 / 9;"` on `Media`  as used in the svelte demo for the home page [vidstack.io](https://www.vidstack.io) 
https://github.com/vidstack/player/blob/3d9ee3129089b09974110760aa17cfc6079cca90/apps/site/src/pages/index/Player.svelte#L95

or  the [`AspectRatio`](https://www.vidstack.io/docs/player/react/components/ui/aspect-ratio/) component?

**Comment by mihar-22** [comment, marked as answer]
Either works but `aspect-ratio` is a [newer feature](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#browser_compatibility) which has less widespread support. I wasn't too concerned with the home page because it's temporary so I just went with `aspect-ratio`.

One reason `<AspectRatio>` sucks today is because it can't be rendered server-side, so layout shifts will occur. This will hopefully be resolved post-tooling upgrade and we can recommend it much more strongly in docs.

**Reply by pietrop** [reply]
Awesome, thanks!
